docs/en/siem/rules-api-create.asciidoc

```diff
@@ -41,19 +84,21 @@ time).
 is converted from a third-party security solution. |No, automatically created 
 when it is not provided.
 
-|index |String[] |Indices on which the rule functions. |No, defaults to the 
-{siem-soln} indices defined on the {kib} Advanced Settings page (*Kibana* → 
-*Management* → *Advanced Settings* → `siem:defaultIndex`).
+|index |String[] |Indices on which the rule functions. |No, for query rules,
+defaults to the {siem-soln} indices defined on the {kib} Advanced Settings page
+(*Kibana* → *Management* → *Advanced Settings* → `siem:defaultIndex`).
 
 |interval |String |Frequency of rule execution, using a
 {ref}/common-options.html#date-math[date math range]. For example, `"1h"` 
 means the rule runs every hour. |No, defaults to `5m` (5 minutes).
 
+|machine_learning_job_id |String |{ml} job ID of the job for which you want to generate signals. |Yes, for {ml} rules.
+
 |query |String |{kibana-ref}/search.html[Query] used by the rule to create a 
-signal. |No, defaults to an empty string.
+signal. |No, for query rules, defaults to an empty string.
```

Comment:
Not sure if it makes sense to capture this here, but just want to mention that certain fields are forbidden depending on the rule type. For instance, for `type: "machine_learning"`, noth `query` and `language` fields are forbidden. It may be a little hard to read, but explicit details are in the [create_rules_schema](https://github.com/elastic/kibana/blob/8f1e22f07852123efc9c8b2946578b2745ef5d47/x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/create_rules_schema.ts#L63-L71) if necessary.

Reply:
Thanks @spong and @rylnd 
Yep - definitely needs correcting